airbyte-webapp/src/packages/cloud/components/experiments/SpeedyConnection/CountDownTimer/CountDownTimer.module.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 .countDownTimerItem {
-  color: colors.$orange;
+  color: colors.$blue-400;
```

Comment:
Sorry I missed that when changing the design, only changed the banner colors.

Reply:
No problem!